Taskfile.yml

```diff
@@ -83,6 +83,22 @@ tasks:
       - task set-user-roles
     silent: true
 
+  import-translations:
+    desc: "Import string and config translations from .po files"
+    prompt: "This will reset all translations to match .po files. Continue?"
+    cmds:
+      - task drush -- locale:import da ../translations/strings/ai-screening.da.po
+      - task drush -- config_translation_po:import da ../translations/config/ai-screening-config.da.po
+      - task drush -- cache:rebuild
+    silent: true
+
+  export-translation-strings:
+    desc: "Export string translations from .po files"
+    prompt: "This will update the .po file in /translations/strings folder"
+    cmds:
+      - task drush -- locale:export da > translations/strings/ai-screening.da.po
+    silent: true
+
```

Comment:
Please tell a bit out these commands and how we handle config translations in the documentation.

How do we edit `ai-screening-config.da.po`?

Reply:
There is no export drush command yet, sp that will have to be handled manually